api/server.py

```diff
@@ -199,24 +243,24 @@ def decorated(*args, **kwargs):
                 raise AuthError(
                     {"code": "token_expired", "description": "token is expired"}, 401
                 )
-            except jwt.JWTClaimsError:
+            except jwt.JWTClaimsError as e:
                 raise AuthError(
                     {
                         "code": "invalid_claims",
-                        "description": "incorrect claims,"
-                        " please check the audience and issuer",
+                        "description": "incorrect claims, please check the audience and issuer",
```

Comment:
could this also include e, or is always this description when a JWTClaimsError is thrown?

Reply:
I thought that one, which only is thrown on a `jwt.JWTClaimsError`exception, would be specific enough - but we can add it if you think it's helpful.

Reply:
I couldn't remember if this was the error we were debugging together, where the error message didnt quite line up exactly with this description